templates/ingress.yaml

```diff
@@ -53,17 +49,67 @@ spec:
             backend:
               serviceName: {{ .Release.Name }}-webserver
               servicePort: airflow-ui
-          - path: /{{ .Release.Name }}/flower
-            backend:
-              serviceName: {{ .Release.Name }}-flower
-              servicePort: flower-ui
     - host: {{- include "airflow_subdomain" . | indent 1 }}
       http:
         paths:
           - path: /
             backend:
               serviceName: {{ .Release.Name }}-webserver
               servicePort: airflow-ui
+{{- end }}
+{{- if and .Values.ingress.enabled (eq .Values.executor "CeleryExecutor") }}
+---
+kind: Ingress
+apiVersion: extensions/v1beta1
+metadata:
+  name: {{ .Release.Name }}-flower-ingress
+  labels:
+    tier: airflow
+    component: airflow-ingress
```

Comment:
@andriisoldatenko maybe `flower-ingress` here?

Reply:
Also, if we change this to `flower-ingress` do we need do do anything on our Grafana or Prometheus stack? I feel like we may have some stuff that looks for this label, but not sure.

Reply:
@schnie 
quick look in `github.com/astronomer/astronomer`
```
$ grep --exclude-dir={.git,.idea} -rnw . -e 'airflow-ingress'
```
return nothing.

Reply:
fixed component name.